### PR TITLE
Schema based on diagram

### PR DESCRIPTION
**Pull request by Tamunokuro**
- Create schema_based_on_diagram.sql file
- Create tables for patients, invoices, treatments, medical_histories and invoice_items

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    unit_price DECIMAL(2,2) NOT NULL,
+    quantity INT NOT NULL,
+    total_price DECIMAL(6,2) NOT NULL
+);
+
+ALTER TABLE invoice_items
+ADD FOREIGN KEY (invoice_id) REFERENCES invoices (id);
+
+ALTER TABLE invoice_items
```
Comment (thecodechaser) [inline schema_based_on_diagram.sql:43]: - [ ] Please make sure to create the "join table" for the many-to-many relationships as requested in the project requirements.

image

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+CREATE TABLE invoice_items (
+    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    unit_price DECIMAL(2,2) NOT NULL,
+    quantity INT NOT NULL,
+    total_price DECIMAL(6,2) NOT NULL
+);
+
+ALTER TABLE invoice_items
+ADD FOREIGN KEY (invoice_id) REFERENCES invoices (id);
+
```
Comment (thecodechaser) [inline schema_based_on_diagram.sql:42]: - [ ] Please make sure all FOREIGN KEYS have indexes created as requested in the project requirements.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Tamunokuro, and @BryteLitty👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**